preview/MsixCore/MsixMgrWix/de-de/strings.wxl

```diff
@@ -1,4 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <WixLocalization Culture="de-de" xmlns="http://schemas.microsoft.com/wix/2006/localization">
-	<String Id="OrphanedPackageWarning">Warnung: Wenn Sie dieses Element entfernen, können die folgenden Apps nicht mehr entfernt werden: [MSIXMGR_PRODUCTS]. Diese Apps sollten entfernt werden, bevor Sie den Vorgang fortsetzen.</String>
+	<String Id="OrphanedPackageWarning">
+		Warnung: Das Entfernen dieses Produkts führt dazu, dass die folgenden App(s), die dadurch installiert werden, nicht mehr entfernt werden können: [MSIXMGR_PRODUCTS]. Diese App(s) sollten vor dem Fortfahren entfernt werden.</String>
```

Comment:
The new translation is also quite close to the English source text, however it is overly complex. I have one little nit about the tense in the main sentence; it should use the past tense. So, instead of “werden” it should be “wurden”. I guess this is either just a typo or the translator had some trouble understanding the full semantics of this particular message.

`Warnung: Das Entfernen dieses Produkts führt dazu, dass die folgenden App(s), die dadurch installiert wurden, nicht mehr entfernt werden können: [MSIXMGR_PRODUCTS]. Diese App(s) sollten vor dem Fortfahren entfernt werden.`

I do not want to lecture anybody but personally, I do not like “overusing” the dative case on verbs, like in the second sentence. This is something one should generally try to avoid in German when writing. One may use this form but only if it helps to convey an idea or concept more easily than a grammatically fully correct sentence. This style is often used by learners of German without knowing any better. In other words, native German authors and writers would probably use true nouns instead.

Anyhow, I think it would be best to replace the occurrence of `this` by the `[ProductName]` substitute for better understanding.

`Warning - uninstalling [ProductName] will cause the following app(s) installed by [ProductName] to become uninstallable: [MSIXMGR_PRODUCTS]. These app(s) should be removed before proceeding.`